src/ledger/schema.zig

```diff
@@ -63,7 +64,6 @@ pub const schema = struct {
     pub const address_signatures: ColumnFamily = .{
         .name = "address_signatures",
         .Key = struct {
-            // NOTE: rn we sort by pubkey first, maybe we want to sort by slot first?
```

Comment:
This column family is used when querying for all signatures from a particular address over a particular timespan. It's a lot more efficient to filter first by address. If it's sorted by slot first, it means the vast majority of items you iterate over will be discarded.